components/Contact.tsx

```diff
@@ -20,18 +32,20 @@ const Contact = () => {
           <div className="col-span-3 lg:col-span-2 w-full h-full shadow-xl shadow-gray-400 rounded-xl p-4 ">
             <div className=" lg:p-3 h-full ">
               <div>
-                <img
+                <Image
                   className=" rounded-xl hover:scale-105 ease-in duration-300 "
-                  src="contact assets/contactpic.jpg"
+                  src="/contact assets/contactpic.jpg"
```

Comment:
Don’t use spaces in web

Also it’ll be nice to have all assets in one folder

Reply:
its already in one folder 
all these folders is in Assets folder